pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -547,8 +548,15 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 		}
 
 		for _, val := range api.RegisteredDevices {
-			if devices, ok := node.Others[val].(api.Devices); ok {
-				code, msg, err := devices.FilterNode(task.Pod)
+			if dev, ok := node.Others[val].(api.Devices); ok {
```

Comment:
Other place in this file like https://github.com/volcano-sh/volcano/blob/2efe281384f2ea3009ae4286480ade11ea514fc7/pkg/scheduler/plugins/predicates/predicates.go#L253 also has this assert, do we also need check wheher it's nil?